src/commands/create_command.py

```diff
@@ -1,32 +1,42 @@
-import json, os
+import json
+import os
 from commands.abstract_command import *
-from commands.keys import Keys
-from output.exceptions import *
+from model.table_metadata import *
 
 
 class CreateCommand(AbstractCommand):
-    def __init__(self, schema_file):
-        self.validate(schema_file, Keys.SCHEMA_PATH)
-        schema_file = os.path.join(Keys.SCHEMA_PATH, schema_file)
-        with open(schema_file, 'r') as file:
-            self.data = json.load(file)
-        self.check_database()
+    def __init__(self, schema_name):
+        self.validate(schema_name, Keys.SCHEMA_PATH)
```

Comment:
Why `validate` is an instance method not a static method ?

Reply:
It is static